### PR TITLE
OpenSlides 3 tests

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
@CatoTH Thanks for you work. We will fix the tests here and if everything is right (with empty tests) we will merge this, so then we can add tests during development.

### CODE REVIEW

**Comment by ostcar** [comment]
Is it possible in travis to run the js-tests in a different test (like the different test for python 3.6 and python 3.7)? So a python change does not have to wait for js-tests?

File: client/src/app/core/services/data-store.service.ts [hunk 1]
```diff
@@ -115,12 +110,7 @@ export class DataStoreService {
      * Empty constructor for dataStore
      * @param cacheService use CacheService to cache the DataStore.
      */
-    public constructor(private cacheService: CacheService, private modelMapper: CollectionStringModelMapperService) {
-        if (DataStoreService.wasInstantiated) {
-            throw new Error('The Datastore should just be instantiated once!');
-        }
-        DataStoreService.wasInstantiated = true;
-    }
+    public constructor(private cacheService: CacheService, private modelMapper: CollectionStringModelMapperService) {}
```
[the remark this answers is not in the document]
Comment (FinnStutzenstein) [reply client/src/app/core/services/data-store.service.ts]: Yes I'm sure.

File: client/src/app/shared/components/search-value-selector/search-value-selector.component.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { Displayable } from '../../models/base/displayable';
 import { TranslateService } from '@ngx-translate/core';
 import { Identifiable } from '../../models/base/identifiable';
 
-type Selectable = Displayable & Identifiable;
+export type Selectable = Displayable & Identifiable;
```
[the remark this answers is not in the document]
Comment (FinnStutzenstein) [reply client/src/app/shared/components/search-value-selector/search-value-selector.component.ts]: Yes, should be done, if it's needed.

[the remark this answers is not in the document]
Comment (jsaalfeld) [reply client/src/app/shared/components/search-value-selector/search-value-selector.component.ts]: You need it in search-value-selector.component.spec.ts line 3.

[the remark this answers is not in the document]
Comment (FinnStutzenstein) [reply client/src/app/shared/components/search-value-selector/search-value-selector.component.ts]: Still, I would create a file next to the component, which is not usefull there. Can be moved, if needed somewhere else except this component.

**Pull request merged**